From ec2d7eba314679e564b051ee497544d7d987b320 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <perex@perex.cz>
Date: Wed, 19 Aug 2020 16:15:04 +0200
Subject: [PATCH 47/51] ucm2: module - rename rk3399-gru-sound.conf =>
 snd_soc_rk3399_gru_sound.conf

We are using the real kernel module name for the first level selection now.

Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---
 ucm2/module/rk3399-gru-sound.conf         | 1 -
 ucm2/module/snd_soc_rk3399_gru_sound.conf | 1 +
 2 files changed, 1 insertion(+), 1 deletion(-)
 delete mode 120000 ucm2/module/rk3399-gru-sound.conf
 create mode 120000 ucm2/module/snd_soc_rk3399_gru_sound.conf

diff --git a/ucm2/module/rk3399-gru-sound.conf b/ucm2/module/rk3399-gru-sound.conf
deleted file mode 120000
index 9116f7731e1d..000000000000
--- a/ucm2/module/rk3399-gru-sound.conf
+++ /dev/null
@@ -1 +0,0 @@
-../Rockchip/rk3399-gru-sound/rk3399-gru-sound.conf
\ No newline at end of file
diff --git a/ucm2/module/snd_soc_rk3399_gru_sound.conf b/ucm2/module/snd_soc_rk3399_gru_sound.conf
new file mode 120000
index 000000000000..9116f7731e1d
--- /dev/null
+++ b/ucm2/module/snd_soc_rk3399_gru_sound.conf
@@ -0,0 +1 @@
+../Rockchip/rk3399-gru-sound/rk3399-gru-sound.conf
\ No newline at end of file
-- 
2.16.4

